swagger/v2/prosecution_case.json

```diff
@@ -205,11 +215,11 @@
     "prosecution_case_reference": {
       "readOnly": true,
       "example": "TFL12345",
-      "description": "The prosecuting authorities reference for their prosecution case that is layed before court.  For example PTI-URN from police/cps cases",
+      "description": "The prosecuting authorities' reference for their prosecution case that is layed before court.  For example PTI-URN from police/cps cases",
```

Comment:
I think this should be `authority's` - Also, could delete the second space between sentences, and I think laid is the more normal spelling not layed

Reply:
Good spot - I've corrected that